src/main/java/frc/robot/commands/Drivetrain/AutoCommand.java

```diff
@@ -0,0 +1,204 @@
+                    trajectory1,
+                    drivetrain::getPose,
+                    RamseteConfig.ramseteController,
+                    RamseteConfig.feedForward,
+                    RamseteConfig.kDriveKinematics,
+                    drivetrain::getWheelSpeeds,
+                    new PIDController(Constants.KP_DRIVE_VEL, 0, 0),
+                    new PIDController(Constants.KP_DRIVE_VEL, 0, 0),
+                    drivetrain::setVolts,
+                    drivetrain
+            );
+
+
+            drivetrain.resetOdometry(trajectory.getInitialPose());
+
+            addCommands(
+                ramCommand,
+                new ElevatorUp(elevator),
+                grabber.setTrue,
+                grabber.setFalse,
+                new ElevatorDown(elevator),
+                ramCommand3,
+                new InstantCommand(() -> drivetrain.setPercent(0, 0))
+            );
+            break;
+        case "InNOut":
+            trajectory = PathPlanner.loadPath("InNOut", new PathConstraints(3.5, 2));
+
+            PPRamseteCommand ramCommand2 = new PPRamseteCommand(
+                    trajectory,
+                    drivetrain::getPose,
+                    RamseteConfig.ramseteController,
+                    RamseteConfig.feedForward,
+                    RamseteConfig.kDriveKinematics,
+                    drivetrain::getWheelSpeeds,
+                    new PIDController(Constants.KP_DRIVE_VEL, 0, 0),
+                    new PIDController(Constants.KP_DRIVE_VEL, 0, 0),
+                    drivetrain::setVolts,
+                    drivetrain
+            );
+
+
+            drivetrain.resetOdometry(trajectory.getInitialPose());
+
+            addCommands(
+                    ramCommand2,
+                    new InstantCommand(() -> drivetrain.setPercent(0, 0))
+            );
+            break;
+        case "SprayNPraySeq":
+            trajectory = PathPlanner.loadPath("RamAuto", new PathConstraints(3.5, 2));
+
+            PPRamseteCommand ramCommand4 = new PPRamseteCommand(
+                    trajectory,
+                    drivetrain::getPose,
+                    RamseteConfig.ramseteController,
+                    RamseteConfig.feedForward,
+                    RamseteConfig.kDriveKinematics,
+                    drivetrain::getWheelSpeeds,
+                    new PIDController(Constants.KP_DRIVE_VEL, 0, 0),
+                    new PIDController(Constants.KP_DRIVE_VEL, 0, 0),
+                    drivetrain::setVolts,
+                    drivetrain
+            );
+
+            drivetrain.resetOdometry(trajectory.getInitialPose());
+            ShooterCommand sc0 = new ShooterCommand(shooter, 0);
+            ShooterCommand sc1 = new ShooterCommand(shooter, 1);
+            ShooterCommand sc2 = new ShooterCommand(shooter, 2);
+            ShooterCommand sc3 = new ShooterCommand(shooter, 3);
+            ShooterCommand sc4 = new ShooterCommand(shooter, 4);
```

Comment:
It's ok to declare these inside addCommands, they don't need to be done with extra variables